autoPyTorch/constants.py

```diff
@@ -0,0 +1,40 @@
+TABULAR_CLASSIFICATION = 1
+IMAGE_CLASSIFICATION = 2
+TABULAR_REGRESSION = 3
+IMAGE_REGRESSION = 4
+
+REGRESSION_TASKS = [TABULAR_REGRESSION, IMAGE_REGRESSION]
+CLASSIFICATION_TASKS = [TABULAR_CLASSIFICATION, IMAGE_CLASSIFICATION]
+
+TASK_TYPES = REGRESSION_TASKS + CLASSIFICATION_TASKS
+
+TASK_TYPES_TO_STRING = \
+    {TABULAR_CLASSIFICATION: 'tabular_classification',
+     IMAGE_CLASSIFICATION: 'image_classification',
+     TABULAR_REGRESSION: 'tabular_regression',
+     IMAGE_REGRESSION: 'image_regression'}
+
+STRING_TO_TASK_TYPES = \
+    {'tabular_classification': TABULAR_CLASSIFICATION,
+     'image_classification': IMAGE_CLASSIFICATION,
+     'tabular_regression': TABULAR_REGRESSION,
+     'image_regression': IMAGE_REGRESSION}
```

Comment:
I think we can later enhance the one for images, like adding detection and so on. I am happy with the tabular ones, which will let us test the pipeline.